packages/babel-plugin-syntax-pipeline-operator/src/index.js

```diff
@@ -1,11 +1,17 @@
 import { declare } from "@babel/helper-plugin-utils";
 
-export default declare(api => {
+const proposals = ["minimal"];
+
+export default declare((api, { proposal }) => {
   api.assertVersion(7);
 
+  if (typeof proposal !== "string" || !proposals.includes(proposal)) {
+    throw new Error("'proposal' must be one of: " + proposals.join(", "));
```

Comment:
Yep looks good, wonder if we should introduce a url to show what the options are and why etc.

Reply:
I also plan on updating the README. Is there any other place where we should include that documentation?

Reply:
I think the readme is good. We should introduce a new general doc on the page explaining how we want to think about proposals moving forward and how they are implemented though. Kinda like https://github.com/babel/proposals/pull/26 but I didn't get to finishing it, not sure.